---
name: Bug report
about: Thanks for helping us improve Polynote!
title: ''
labels: ''
assignees: ''

---

**Description**
A clear and concise description of what the bug is.

**Reproduction**
Steps to reproduce the behavior:
1. ...
2. ...

**Environment**
Are you using the official Docker distribution? If so, please provide your `docker run` command and any other details that may be relevant (mount points, etc). 

Otherwise, please provide the following information:
- Polynote Version: 
   - Example: 0.2.12
- Operating System: 
  - Example: Mac OS Mojave 10.14.6
- JVM Version:
  - Example: 1.8.0_212
- Python Version:
  - Example: 2.7.15
- How you installed Polynote: 
  - Example: I downloaded this release https://github.com/polynote/polynote/releases/download/0.2.12/polynote-dist.tar.gz and then followed [these instructions](https://polynote.org/docs/01-installation.html).
- The command you are using to start Polynote:
  - Example: `./polynote.py`
- Any other details that might be useful, such as whether you are running Polynote as a service, using a custom script, etc.  

**Configuration**
If applicable, please attach or paste your Polynote configuration (`config.yml`)

**Notebook**
If applicable, please attach or paste the notebook file (`.ipynb`) you were trying to run when you see this bug. 

**Logs**
If applicable, please attach or paste the Polynote log

**Screenshots**
If applicable, please attach screenshots to help explain your problem.

**Additional context**
Add any other context about the problem here.
